go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/ethereum/go-ethereum v1.10.11
+	github.com/guregu/null v3.5.0+incompatible
```

Comment:
Should this be `gopkg.in/guregu/null.v4`?

Reply:
yeah idk why it keeps trying to make me use 3.5.0...

Reply:
oh I see, it's the wrong package. `gopkg.in/guregu/null` vs `github.com/guregu/null`